src/lib/components/Search.svelte

```diff
@@ -71,25 +71,25 @@
         {
             uid: 'recommended-references-account',
             url: '/docs/references/cloud/client-web/databases',
-            h1: 'References',
+            h1: 'API Reference',
```

Comment:
```suggestion
            h1: 'API reference',
```

We use sentence case! rest looks good.